site/docs/v1/tech/core-concepts/tenants.adoc

```diff
@@ -147,6 +163,15 @@ The email template to use when accounts are created to verify the User's email a
 +
 Required when the [field]#Verify email# toggle is enabled.
 
+[field]#Verification strategy# [optional]#Optional#::
```

Comment:
This doesn't have the `since` marker. Should it? Should the others in this section?